### ISSUE TITLE
Pac3 Permanent Trail

### ISSUE THREAD

**Issue by StinkyWeeb** [CapsAdmin/pac3#467]
I was first trying to add a weapon item when this bug surfaced. I had loaded into a new game and for some odd reason a trail that was meant for the projectile was showing on my character, which was strange considering the item was hidden by hiding the group it was in. I tried clearing my Pac to see if it was just a bad Pac to no avail, the trail persisted. I tried then restarting my game, and once again, it persisted. Then I tried applying a different playermodel and clearing the Pac again in the new game. Nothing, and once again, it was still there.

**Tl;Dr -- A trail in the default position won't come off no matter what I try.**
